README.md

```diff
@@ -19,3 +19,17 @@ python3 -m unittest tests
 # Run a specific test
 python3 -m unittest tests/test_basic
 ```
+
+Badges:
+```sh
+# Version Badge
+.. image:: https://badge.fury.io/gh/marco-c%2Fcrashsimilarity.svg
```

Comment:
See the result: https://github.com/aditya-iitd/crashsimilarity/blob/8f23b20cc843b57476718bd60493f451d15ca227/README.md